octodns/provider/transip.py

```diff
@@ -1,18 +1,17 @@
-#
-#
-#
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
 
-from __future__ import absolute_import, division, print_function, \
-    unicode_literals
+from collections import defaultdict, namedtuple
+from logging import getLogger
 
-from suds import WebFault
+from transip import TransIP
+from transip.exceptions import TransIPHTTPError
+from transip.v6.objects import DnsEntry
 
-from collections import defaultdict
-from .base import BaseProvider
-from logging import getLogger
 from ..record import Record
-from transip.service.domain import DomainService
-from transip.service.objects import DnsEntry
+from .base import BaseProvider
+
+DNSEntry = namedtuple("DNSEntry", ("name", "expire", "type", "content"))
```

Comment:
octoDNS prefers single quotes over double. They're used in pretty much all cases except when `'` is in the string.

Looks like there's quite a bit of reformatting changes in this PR. That makes it a bit harder to tell what's actually changed and what's been reformatted, nearly 90% of lines appear to have changes. In the future I'd recommend not combining formatting changes in a PR with meaningful changes and more generally recommend following the prevailing style.